PostmanCollections/VTEX - Marketplace APIs - Sent Offers.json

```diff
@@ -1,23 +1,23 @@
+              "content": ">ℹ Info\r\n>\r\n>To evolve your experience in synchronizing and updating offers, the VTEX team is developing a new functionality that will replace the current module <b>Offer Management</b> . As a result, the Offer Management module will not receive new updates or maintenance. \r\n>\r\n>For connectors and marketplaces already integrated into the module, please contact the development team by email [taissa.araujo@vtex.com.br](taissa.araujo@vtex.com.br). \r\n>\r\n>For connectors interested in starting the integration, we ask you to wait for the release of the new module, as soon as it is available we will announce it to all customers through the [dev portal](https://developers.vtex.com/updates/release-notes) and [help VTEX](https://help.vtex.com/pt/en/announcements). \r\n\r\n The first step for connectors to integrate with [Offer Management](https://developers.vtex.com/vtex-rest-api/docs/sent-offers-integration-guide-connectors) is to create a channel, that represents the marketplace to where sellers will send their offers. \n\nThis endpoint creates a channel and the integration developers should call it once for each marketplace. The information about the marketplace sent in the request will be shown to sellers in their [Offer Management UI](https://help.vtex.com/en/tutorial/offers-listing--7MRb9S78aBdZjFGpbuffpE). \n\nThe `feedId` created by this call will apply to all sellers connected to the given channel, and will be necessary for the next step of the integration flow, which is to [activate feed](https://developers.vtex.com/docs/api-reference/marketplace-apis-offer-management#post-/api/sent-offers/feeds). \n\n>ℹ Info\r\n> \r\n> Offer Management is available for integrations with [Mercado Livre (Classic and Premium)](https://help.vtex.com/pt/tracks/configurar-integracao-do-mercado-livre--2YfvI3Jxe0CGIKoWIGQEIq), [Netshoes](https://help.vtex.com/pt/tracks/configurar-integracao-da-netshoes--5Ua87lhFg4m0kEcuyqmcCm), and VTEX marketplaces. For more information, see [Offer Management Integration Guide](https://developers.vtex.com/vtex-rest-api/docs/sent-offers-integration-guide-connectors).",
```

Comment:
Esse callout ficaria renderizado como o da foto abaixo, mas substituindo "Onboarding guide" por `Info`

**Exemplo 1**
O [código](https://developers.vtex.com/api/openapi/antifraud-provider-protocol) fica assim:
```
>ℹ️ Onboarding guide\r\n>\r\n> Check the new [Payments onboarding guide](https://developers.vtex.com/docs/guides/payments-overview). We created this guide to improve the onboarding experience for developers at VTEX. It assembles all documentation on our Developer Portal about Payments and is organized by focusing on the developer's journey.\r\n\r\n
```

<img width="673" alt="image" src="https://github.com/vtex/openapi-schemas/assets/83723439/9db05b25-192c-470c-b44a-e6c8e54e717e">


**Exemplo 2**
Outros exemplos deixam esse código do callout mais limpo:

```
\r\n> Check the new [Catalog onboarding guide](https://developers.vtex.com/docs/guides/catalog-overview). We created this guide to improve the onboarding experience for developers at VTEX. It assembles all documentation on our Developer Portal about Catalog and is organized by focusing on the developer's journey.\r\n\r\n
```

Foto de como fica renderizado:
<img width="532" alt="image" src="https://github.com/vtex/openapi-schemas/assets/83723439/5c2819ec-a8b7-4b60-9541-3ca062d90414">


**Meus two cents:**
- No seu caso, já que já tem o ícone de ℹ️ , acho desnecessário trazer esse título "info" repetindo a palavra. Então removeria, e passaria direto pro texto *To evolve your experience in synchronizing and updating offers [...]*